README.md

```diff
@@ -160,8 +160,10 @@ Default directory structure
   - `controllers`: folder for controller files.
   - `models`: folder for model files.
   - `views`: folder for view files.
+- `lib`: Optional folder for your application's utility libraries.
```

Comment:
Rewrite this to "Optional folder for any modules, utility libraries, or other dependencies your app may require which aren't in npm, symlinked to node_modules."